CHANGES.rst

```diff
@@ -2,11 +2,13 @@
 History
 =======
 
-1.3.1
+2.0
```

Comment:
To make sure `zest.releaser` finds this line it would be nice if it reads:
`2.0 (unreleased)`